src/Microsoft.DotNet.Build.Tasks/Targets/tests.targets

```diff
@@ -22,8 +22,9 @@
 
   <!-- General xunit options -->
   <PropertyGroup>
+    <XunitResultsFileName>testResults.xml</XunitResultsFileName>
     <XunitCommandLine>xunit.console.netcore.exe $(TargetFileName)</XunitCommandLine>
-    <XunitOptions>$(XunitOptions) -xml .\testResults.xml</XunitOptions>
+    <XunitOptions>$(XunitOptions) -xml $(XunitResultsFileName)</XunitOptions>
```

Comment:
What conflict will we have? If I understand correctly, batching will occur based on the target framework, which is also part of the test working directory.